src/python/python-proto/python_proto/pipeline.py

```diff
@@ -86,7 +86,8 @@ def into_proto(self) -> _Envelope:
         proto_envelope.metadata.CopyFrom(self.metadata.into_proto())
         inner_message = _Any()
         inner_message.Pack(
-            self.inner_message.into_proto(), type_url_prefix="".encode("utf-8")
+            self.inner_message.into_proto(),
+            type_url_prefix="graplsecurity.com".encode("utf-8"),
```

Comment:
You could also just do b"graplsecurity.com"

Reply:
I'm not 100% sure these things are actually identical? A `b""` is a bytes literal but `"".encode("utf-8")` is saying "encode my string in UTF-8 bytes". That is, the important thing here is the encoding. While I _suspect_ this source file is UTF-8 encoded, I'm not sure whether that's specified anywhere?

Reply:
hmm there's some more clarity here https://docs.python.org/3/reference/lexical_analysis.html#string-and-bytes-literals

in particular,

> The source character set is defined by the encoding declaration; it is UTF-8 if no encoding declaration is given in the source file; see section [Encoding declarations](https://docs.python.org/3/reference/lexical_analysis.html#encodings).

and

> Bytes literals are always prefixed with `'b'` or `'B'`; they produce an instance of the [`bytes`](https://docs.python.org/3/library/stdtypes.html#bytes) type instead of the [`str`](https://docs.python.org/3/library/stdtypes.html#str) type. They may only contain ASCII characters; bytes with a numeric value of 128 or greater must be expressed with escapes.

So, given that ASCII is a subset of UTF-8 a `b".."` here is equivalent to `"..".encode("utf-8")`. Will fix.

Reply:
fixed